src/main/java/com/btkelly/gnag/models/ViolationComment.java

```diff
@@ -25,7 +25,7 @@
 
     public ViolationComment(boolean failBuild, String commentMessage) {
```

Comment:
If you were awesome, this would read

    public ViolationComment(boolean failBuild, @NonNull String commentMessage)

or

    public ViolationComment(final boolean failBuild, @NonNull final String commentMessage)